state-chain/pallets/cf-trading-strategy/src/lib.rs

```diff
@@ -56,7 +56,7 @@ impl_pallet_safe_mode!(PalletSafeMode; strategy_updates_enabled, strategy_closur
 	Clone, Debug, Encode, Decode, TypeInfo, serde::Serialize, serde::Deserialize, PartialEq, Eq,
 )]
 pub enum TradingStrategy {
-	SellAndBuyAtTicks { sell_tick: Tick, buy_tick: Tick, base_asset: Asset },
+	SellAndBuyAtTicks { spread_tick: Tick, base_asset: Asset },
```

Comment:
Maybe we should reserve this name for a potential future strategy where you can configure sell and buy ticks individually (as before)? How about `TickZeroCentered` for this one?